cms_status_message/README.rst

```diff
@@ -0,0 +1,97 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+CMS status message
+==================
+
+A "status message" is an important message that you want to show to users.
+
+For instance: a user submit a form or does a specific action
+and you want to report the status of this action
+like "your profile has been updated" or "Your upgrade has been successful.".
+
+This module allows to easily display this kind of messages to your users.
+
+Messages are displayed using Twitter bootstrap alerts.
+
+You can add several messages: they will be displayed one after another.
+
+Usage
+=====
+
+Add status message to layout::
+
+    <template id="add_status_message" inherit_id="website.layout" name="Add status message">
+
+        <xpath expr="//div[@id='wrapwrap']/main/*" position="before">
+            <t t-call="cms_status_message.status_message" />
+        </xpath>
+
+    </template>
```

Comment:
was my 1st idea then I decided to not decide for everyone... but it makes sense by default ;P DONE!